proto/babylon/finality/v1/query.proto

```diff
@@ -146,11 +146,22 @@ message QueryBlockRequest {
   uint64 height = 1;
 }
 
+// BlockResponse is the necessary metadata and finalization status of a block
+message BlockResponse {
```

Comment:
I think I would name it `IndexedBlockResponse` to make it clear that the response contains indexed blocks